specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-03-01/managedClusters.json

```diff
@@ -2281,7 +2281,7 @@
           "description": "Number of agents (VMs) to host docker containers. Allowed values must be in the range of 0 to 100 (inclusive) for user pools and in the range of 1 to 100 (inclusive) for system pools. The default value is 1."
         },
         "vmSize": {
-          "$ref": "#/definitions/ContainerServiceVMSize",
+          "type": "string",
```

Comment:
this should not be breaking as the enum was already model-as-string?

Reply:
and yet, the enum values were modeled in e.g. the Go SDK so this does require a client change...I thought the point of modelAsString to allow for non-breaking changes to enum values?